README.md

```diff
@@ -13,11 +13,13 @@ Learn some math basics! Focus only on these topics, then come back later in case
  * [Khan Academy - Linear Algebra](https://www.khanacademy.org/math/linear-algebra)
  * [Khan Academy - Statistics Probability](https://www.khanacademy.org/math/statistics-probability)
  * [Optional: 3Blue1Brown - Essence of Linear Algebra](https://www.3blue1brown.com/essence-of-linear-algebra-page/)
+ * [Optional - The Organic Chemistry Tutor Math](https://www.youtube.com/@TheOrganicChemistryTutor/playlists)
```

Comment:
This has nothing to do with Machine Learning

Reply:
Removed. Also kindly ignore commit message. My apologies